src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
```

Comment:
The variable name 'result' is too generic. According to the checklist, avoid variable namings like 'result' as it decreases code readability. Consider using a more descriptive name, such as 'styleObject' or 'cssProperties'. [CODE STYLE]